Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -646,6 +648,70 @@ void ADefiWalletCoreActor::SendEthAmount(int32 walletIndex, FString fromaddress,
   }
 }
 
+void ADefiWalletCoreActor::SendEthAmountAsync(
+    int32 walletIndex, FString fromaddress, FString toaddress, FString amount,
+    FString gasLimit, FString gasPrice, TArray<uint8> txdata,
+    FSendEthTransferDelegate Out) {
+
+  AsyncTask(ENamedThreads::AnyHiPriThreadNormalTask, [this, Out, walletIndex,
+                                                      fromaddress, toaddress,
+                                                      amount, gasLimit,
+                                                      gasPrice, txdata]() {
+    FString result;
+    FCronosTransactionReceiptRaw txresult;
+
+    try {
+      if (NULL == _coreWallet) {
+        result = TEXT("Invalid Wallet");
+      } else {
+        std::string mycronosrpc = TCHAR_TO_UTF8(*myCronosRpc); /* 8545 port */
+
+        rust::cxxbridge1::String myaddress1 =
+            _coreWallet->get_eth_address(walletIndex);
+        rust::cxxbridge1::String myaddress2 = TCHAR_TO_UTF8(*toaddress);
+        rust::cxxbridge1::String nonce1 =
+            get_eth_nonce(myaddress1.c_str(), mycronosrpc);
+        char hdpath[100];
+        snprintf(hdpath, sizeof(hdpath), "m/44'/%d'/0'/0/%d", EthCoinType,
+                 walletIndex);
+        rust::cxxbridge1::Box<PrivateKey> privatekey =
+            _coreWallet->get_key(hdpath);
+        rust::cxxbridge1::Vec<uint8_t> data;
+        org::defi_wallet_core::EthTxInfoRaw eth_tx_info = new_eth_tx_info();
+        cout << myaddress2 << endl;
```

Comment:
maybe use that unreal logging macro instead of writing to cout?

Reply:
removed